internal/kms/single-key.go

```diff
@@ -303,6 +303,11 @@ func (kms secretKey) DecryptAll(_ context.Context, keyID string, ciphertexts [][
 	return plaintexts, nil
 }
 
+// Verify verifies all KMS endpoints and returns details
+func (kms secretKey) Verify(cxt context.Context) []VerifyResult {
+	return []VerifyResult{}
+}
+
```

Comment:
you could implement it and indicate the `endpoint` just say that it is "[self]"

Reply:
Ack